### PR TITLE
added readme for algorithms sudoku solver

### PR DESCRIPTION
**Pull request by ycechungAI**
added readme for sudoku solver under algoirthms

need more instructions to add machine learning README

### CODE REVIEW

**Comment by ycechungAI** [comment]
#432 added

File: samples/machine-learning/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Machine Learning Samples
+
+- **[Half Moons](./half-moons)**
+This sample implements using the quantum machine learning library to train a sequential model on the half-moons dataset.
```
Comment (msoeken) [inline samples/machine-learning/README.md:4]:
```suggestion
- **[Half Moons](./half-moons)**
This sample demonstrates how to use the quantum machine library to train a sequential model on the half-moons dataset.
```

File: samples/machine-learning/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+- **[Parallel Half Moons](./parallel-half-moons)**
+This sample implements using the quantum machine learning library to train a sequential model on the half-moons dataset, parallelizing over target machines
```
Comment (msoeken) [inline samples/machine-learning/README.md:7]:
```suggestion
- **[Parallel Half Moons](./parallel-half-moons)**
This sample demonstrates how to use the quantum machine library to train a sequential model on the half-moons dataset, parallelizing over target machines.
```

**Review by msoeken — CHANGES_REQUESTED** [review]
Thanks for your contribution @ycechungAI, looks good to me after a few suggestions.

**Review by msoeken — APPROVED** [review]

**Review by ycechungAI — COMMENTED** [review]
not sure about these warnings for unknown word netcoreapp, but shouldnt affec't the readme.

File: samples/algorithms/README.md [hunk 3]
```diff
@@ -14,3 +16,4 @@
   This sample demonstrates how to build quantum oracles from reversible classical logic.
 - **[Order Finding](./order-finding)**:
   This sample uses a quantum algorithm to find the order of a cycle in a permutation.
+
```
Comment (msoeken) [inline samples/algorithms/README.md:19]:
```suggestion
```

**Comment by msoeken** [comment]
One CI error was due to multiple empty lines in Markdown. I fixed this problem in the last commit. Let's await CI.

**Pull request merged**